grakn-graql/src/main/java/ai/grakn/graql/internal/gremlin/sets/DataTypeFragmentSet.java

```diff
@@ -21,16 +21,26 @@
 
 import ai.grakn.concept.AttributeType;
 import ai.grakn.graql.Var;
-import ai.grakn.graql.admin.VarProperty;
 import ai.grakn.graql.internal.gremlin.EquivalentFragmentSet;
+import ai.grakn.graql.internal.gremlin.fragment.Fragment;
 import ai.grakn.graql.internal.gremlin.fragment.Fragments;
+import com.google.auto.value.AutoValue;
+import com.google.common.collect.ImmutableSet;
+
+import java.util.Set;
 
 /**
  * @author Felix Chapman
```

Comment:
Bad java docs ar bad :(

Reply:
I never wrote these because they are already documented in `EquivalentFragmentSets`.

But I'm going to do things properly with a fancy `{@see EquivalentFragmentSets#blah}` thing! No duplication and everyone is happy.

Reply:
I am never happy Felix. Just less miserable and this would indeed make me less miserable :P